lib/game/simple_game.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flame/components/component.dart';
+
+import 'base_game.dart';
+
+/// This is a helper implementation of a [BaseGame] designed to allow to easily create a game with a single component.
+///
+/// This is useful to add sprites, animations and other Flame components "directly" to your non-game Flutter widget tree, when combined with [EmbeddedGameWidget].
+class SimpleGame extends BaseGame {
+  SimpleGame(Component c) {
+    add(c);
+  }
+}
```

Comment:
IMO this should not be on a separate file, just unnecessary indirection.

Reply:
Should it be there at all though..?

Reply:
Move it to `EmbeddedGameWidget`?

Reply:
Although, it think its good to be consistent, like one class per file. Not sure if you have any style guides / rules for these kind of things?

Reply:
I have some doubts about this class as will, I think that could be removed, as just doing the following would replace it

```dart
BaseGame()
  ..add(comp);
```

Reply:
@erickzanardo that class is abstract now, so can't be instantiated, but i could make it non-abstract

Reply:
I pushed a new commit where i remove SimpleGame and make BaseGame non-abstract. Fixed and testet with the Animation example

Reply:
Good solution!